resources/monitoring/templates/kyma-additions/istio-telemetry-prometheus-svc-monitor-patch.yaml

```diff
@@ -0,0 +1,23 @@
+# remove after fix
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istio-telemetry-svc-monitor
```

Comment:
Is that the name of the upstream monitor or self-invented?
If self-invented please align it with the naming of the other monitors of the chart